### ISSUE TITLE
upload failed

### ISSUE THREAD

**Issue by kaiby** [minio/minio#13322]
## Current Behavior

![image](https://user-images.githubusercontent.com/3312664/135219101-a63d5713-c7a6-4bc3-9bdf-ac2c1b9da851.png)

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used (`minio --version`):minio version RELEASE.2021-09-24T00-24-24Z
* Server setup and configuration:
* Operating System and version (`uname -a`): centos 7

**Comment by harshavardhana** [comment]
@kaiby looks like your backend might be not reachable can you enable `_MINIO_SERVER_DEBUG=on`

**Comment by kaiby** [comment]
> @kaiby looks like your backend might be not reachable can you enable `_MINIO_SERVER_DEBUG=on`
how to use,I run command:
`./minio server data --console-address ":9001"`

thanks

**Comment by kaiby** [comment]
![image](https://user-images.githubusercontent.com/3312664/135227559-6bdf9612-c62b-4751-b387-03706e62edb8.png)

**Comment by harshavardhana** [comment]
I did not ask for the trace @kaiby - start the server with  `export _MINIO_SERVER_DEBUG=on`

**Comment by kaiby** [comment]
> export _MINIO_SERVER_DEBUG=on
![image](https://user-images.githubusercontent.com/3312664/135241264-afba5e37-b9db-4e77-b2a6-c34ba396b441.png)

![image](https://user-images.githubusercontent.com/3312664/135241170-eea49b75-0385-4d21-a09a-d05556afabbf.png)

**Comment by kce68** [comment]
Unfortunately exactly the same behaviour here.
None of my S3-clients (mc, console, others) can create oder modify files. Download an delete works.
All began after updating my cluster from 2021-09-15T04-54-25Z to latest (2021-09-24T00-24-24Z) this morning.

Can I roll back safely until this bug is fixed?

Can I be of som help with logfiles before that?

thx & regards,
Kai

**Comment by ryanlelek** [comment]
Issue #13267 may be related based on your descriptions - @kce68 @kaiby
RELEASE.2021-09-24 and RELEASE.2021-09-18 seem to have introduced an issue

**Comment by kce68** [comment]
I have just rolled back to RELEASE.2021-09-18 and everything works fine again. 
My Cluster has 4 physicals with 8 disks each. with TLS.

**Comment by DillFrescott** [comment]
Exact same issue happening to me. Its frustrating to say the least :)

**Comment by harshavardhana** [comment]
Two things I can think could have happened is silent crash due to 

- hard quota usage fixed in
```
commit 02c24a860dbf216e0393b69cec23b5761af575ba
Author: Harshavardhana <harsha@minio.io>
Date:   Mon Oct 11 11:03:54 2021 -0700

    fix: crash in hard quota enforcement (#13403)
```

- another one is disabling TLS client auth by default, which can interfere 
```
commit 9ea45399ceb40e5b70b19e3578da8193ba5f1c2a
Author: Harshavardhana <harsha@minio.io>
Date:   Mon Oct 11 14:23:51 2021 -0700

    fix: enable AssumeRoleWithCertificate API only when asked (#13410)
    
    This is a breaking change but we need to do this to avoid
    issues discussed in #13409 based on discussions from #13371
    
    fixes #13371
    fixes #13409
```

**Comment by kce68** [comment]
> Two things I can think could have happened is silent crash due to
> 

Don't know which on it was:

I did test an update to the latest version today (2021-10-13T00:23:17Z) and everything works fine !!
Didn't change anting within my configuration.

Thx an d Regards, Kai
